package/yast2.changes

```diff
@@ -279,6 +279,11 @@ Fri Oct 26 11:46:16 UTC 2018 - jreidinger@suse.com
   not exist or is empty (bsc#948729)
 - 4.1.32
 
+-------------------------------------------------------------------
+Thu Oct 25 09:37:50 UTC 2018 - Jan Engelhardt <jengelh@inai.de>
+
+- Use noun phrase in summary.
```

Comment:
I am sorry, but this won't work. Changes have to strictly incrementing and no backward inserting is allowed as there are users which check top of changes for new things. So please change date and put it on top or remove this part and use again `osc vc`